### PR TITLE
Problem: existingcontent __getattr__ is piling hacks

### PR DESCRIPTION
**Pull request by gotcha**
Solution: get rid of it since it does not seem to be needed anymore

Add an error message for the corner case that might still break

(I'll make another PR for main branch if this gets accepted.)

### CODE REVIEW

**Comment by gotcha** [comment]
@mauritsvanrees Thanks for reviewing. I'd also appreciate another pair of eyes.

Further, would you mind reviewing the commit I added 1 minute after your review 😬 ?

**Review by jensens — COMMENTED** [review]
Not blockers, but some comments from me.

File: plone/app/standardtiles/existingcontent.py [hunk 1]
```diff
@@ -199,15 +209,25 @@ def item_panels(self):
...
+                for child in panel.getchildren():
+                    # lxml element needs to be copied
+                    # to put it out of context of the root tree it comes from.
+                    # If this is not done, serializer will keep on serializing
+                    # after element is closed until last
+                    # element of the root tree.
+                    child_copy = copy.deepcopy(child)
+                    child_html = safe_unicode(serializer(child_copy))
+                    panel_html.append(child_html)
+                panel_html = "".join(panel_html)
```
[the remark this answers is not in the document]
Comment (gotcha) [reply plone/app/standardtiles/existingcontent.py:226]: Used to be the case. Dunno if I am cargo culting.

**Comment by mauritsvanrees** [comment]
> Further, would you mind reviewing the commit I added 1 minute after your review 😬 ?

Still seems to work. Tested on Plone 5.2.8, Python 3.8, plone.app.mosaic 2.2.3.

**Review by jensens — APPROVED** [review]

**Comment by jensens** [comment]
Merge after adding a changelog entry.

**Pull request merged**